src/run/formlyNgModelAttrsManipulator.js

```diff
@@ -124,7 +124,38 @@ function addFormlyNgModelAttrsManipulator(formlyConfig, $interpolate, formlyWarn
     const selectorNot = angular.isString(skip) ? `:not(${skip})` : '';
     const skipNot = ':not([formly-skip-ng-model-attrs-manipulator])';
     const query = `[ng-model]${selectorNot}${skipNot}, [data-ng-model]${selectorNot}${skipNot}`;
-    return node.querySelectorAll(query);
+    try {
+      return node.querySelectorAll(query);
```

Comment:
Could you add a comment here that all of this can go away when we drop support for IE8? Not going to happen in 7.0.0, but may happen in 8.0.0